src/App.vue

```diff
@@ -1,31 +1,41 @@
 <template>
   <div id="app">
-    <Button title="button" v-bind:color="btnColor.Primary" size="lg" v-bind:is-disabled="true"
-            v-bind:action=" () => showAlert('Hello!')"></Button>
-    <Button title="button" v-bind:color="btnColor.Secondary" size="lg" v-bind:is-disabled="false"></Button>
+    <app-button title="button" v-bind:color="btnColor.Primary" size="lg" v-bind:is-disabled="true"
```

Comment:
```suggestion
    <app-button
      title="button"
      v-bind:color="btnColor.Primary"
      size="lg" v-bind:is-disabled="true"
      v-bind:action=" () => showAlert('Hello!')" />
```
My suggestion is to align attributes. If you have nothing between `<app-button...>` and `</app-button>`, you can shorten it: `<app-button... />`

Reply:
Make sense, thanks!